src/components/navbar/useDownloadPanel.ts

```diff
@@ -58,7 +58,7 @@ const useDownloadPanel = (projectConfig: ProjectConfig) => {
             const a = Object.assign(document.createElement('a'), {
                 href: objectUrl,
                 style: 'display: none',
-                download: `export.${extension}`,
+                download: `${projectConfig.projectName}.${extension}`,
```

Comment:
the `{projectConfig.projectName}` can be passed as empty string, i suggest that you fallback to the project name that we get from `onProjectInfoRequested`

Reply:
Done